### PR TITLE
Please check my two commits

### PR DESCRIPTION
**Pull request by vjascanu**
Hope this will go well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 Repository for trying stuff
 
 [vjascanu] - added some new content.
+[vjascanu ]- added one more line for second commit
```
Comment (njascanu) [inline README.md]: I was expecting some lines of code for second commit

Comment (vjascanu) [reply README.md]: I don't really understand what I need to do

Comment (njascanu) [reply README.md]: Just add a new Python hello world line

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 
+
 # hello-world
 Repository for trying stuff
 
 [vjascanu] - added some new content.
 [vjascanu ]- added one more line for second commit
+
+println("Hello World")
```
Comment (njascanu) [inline README.md]: Still not enough. Add one more line

**Pull request merged**